plans/academic_after_hours.markdown

```diff
@@ -0,0 +1,101 @@
+# Critical Challenges Planning
+
+## The Challenge
+
+* What Critical Challenge are you going to work on?
+
+Coming up with a way to provide effective support outside of "business hours" without overburdening instructional staff.
+
+* Why is this a “challenge”?
+
+It is difficult to maintain productivity without having instructional staff work an absurd amount of horus.
+
+* Can it be solved? What would be value to the organization/team in solving it or working towards a solution?
+
+There are approaches to attempt in order to continue to make the evening be productive.
+
+* What happens if we fail to solve it / improve it?
+
+Students will feel that outside of the day they are on their own.
+
+* What skills do you have that make you the right person to work on this challenge?
+
+Having the Turing experience fresh in my mind.
+
+## The Past
+
+* What’s been done so far?
+
+The most effective so far has been scheduling pairing sessions in the evening post classes.
+
+* Was it effective? How do you know?
+
+1505 has started to arrange more mentor pairing sessions in the evening outside of assigned ones.
+
+* Was this someone’s challenge before you? Why did they stop doing it or how did we make progress without addressing it?
+
+I believe that Jeff was looking into ways to further this.
+
+
+## The Present
+
+* What are the first steps that you’re ready to execute now?
+
+Continue mentor pairing in the evening.
+
+* What will be the next three things you do?
+
+Continue to schedule pairing sessions with 1505. Reach out to our wonderful mentors to get free times and just force all students
+to pair. Perhaps have an "On-Duty Mentor"
+
+* How will you keep track of progress and results?
+
+Measure at the number of people who continue to work at Turing into the evening.
+ 
+* Are you empowered to take the actions you think necessary?
+
+Definitely.
+
+## The Future
+
+* What would total success look like?
+
+A larger number of people staying at Turing to work during the evening.
```

Comment:
Hmm. Maybe also ensure that staff members aren't working crazy hours and reaching potential burn out.